pkg/framework/backoff.go

```diff
@@ -7,7 +7,7 @@ import (
 )
 
 const (
-	LongMaxWait  = 30 * time.Minute
+	LongMaxWait  = 40 * time.Minute
```

Comment:
This is for Azure because it takes super long sometimes.

Reply:
This is sad but necessary I guess.